ush/machine/hera.yaml

```diff
@@ -17,6 +17,7 @@ platform:
   QUEUE_HPSS: batch
   RUN_CMD_FCST: srun --export=ALL
   RUN_CMD_POST: srun --export=ALL
+  RUN_CMD_AQMLBC: srun --export=ALL -n ${NUMTS}
```

Comment:
@padhrigmccarthy, can you move this `RUN_CMD_AQMLBC` line below the `RUN_CMD_AQM` command line (Ln 23)?